projects/go-lib/src/lib/components/go-side-nav/go-side-nav/go-side-nav.component.scss

```diff
@@ -54,10 +54,11 @@
 .go-side-nav__app-icon {
   display: flex;
   font-size: 1.5rem;
-  padding: 0 1rem 0 .5rem;
+  padding: .2rem 1rem 0 .3rem;
 }
 
 .go-side-nav__app-drawer {
+  border-radius: 50%;
```

Comment:
I don't think we need this anymore if it was switched to the `go-icon-button` right?

Reply:
True, removing it in latest commit. Thanks!